x-pack/platform/plugins/shared/ml/public/cases/constants.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+/**
+ * Titles for the cases toast messages
+ */
+export const TITLES = {
```

Comment:
Again, suggest to make this const more specific like `CASES_TOAST_MESSAGES_TITLES`.

Reply:
Done in: [#7f47fad](https://github.com/elastic/kibana/pull/203813/commits/7f47fad4323c3b3555c29d229b95e093bd1eadb8)